03-CSS/src/css/jungle/jungle-variables.css

```diff
@@ -0,0 +1,9 @@
+
+
+:root {
+    --text-color: rgb(9, 51, 21);
+    --dirty-olive: rgb(14, 83, 41);
+    --brown: rgb(12, 114, 51);
+    --dark-brown: rgb(70, 131, 20);
+    --green: #b48135;
```

Comment:
Wenn du die inhalte der Variablen stark änderst macht es Sinn sie auch umzubenennen.
Mit rechtsklick auf den namen > `rename symbol` kannst du alle Instanzen der Variablen in einer Datei umbenennen.

Reply:
Macht Sinn. ^^